microsoft/testsuites/power/power.py

```diff
@@ -254,7 +254,7 @@ def verify_hibernation_max_data_disks(self, node: Node, log: Logger) -> None:
         data_disks_after_hibernation = disk.get_raw_data_disks()
         assert_that(data_disks_before_hibernation).described_as(
             "data disks are inconsistent after hibernation"
-        ).is_length(data_disks_after_hibernation)
+        ).is_length(len(data_disks_after_hibernation))
```

Comment:
is_length is enough?

Reply:
Int is expected as argument 
![image](https://github.com/user-attachments/assets/47570403-08f4-479a-89d1-e13d37a40817)